problems/stack/stack.py

```diff
@@ -5,13 +5,19 @@ class Stack():
     def __init__(self):
         self.stack = []
 
+    def isEmpty(self):
+		return True if len(self.stack) == 0 else False
+
     def push(self, i):
         self.stack.append(i)
 
     def pop(self):
+    	if(self.isEmpty()):
+			return 'Underflow'
```

Comment:
I suggest to return `None`, here. This allows using the return value in a condition like so:

```python
s = Stack()
val = s.pop()
if val:
  # Do something with val here
  pass
```

[Starting with Python 3.8](https://www.python.org/dev/peps/pep-0572/), you can also write the following:

```python
if (val := s.pop()) is not None:
  # Do something with val
  pass  
```